csi/volumeutils.py

```diff
@@ -1294,8 +1294,6 @@ def mount_glusterfs_with_host(volname, mountpoint, hosts, options=None, is_clien
         for option in options.split(","):
             g_ops.append(f"--{option}")
 
-    cmd.append(mountpoint)
```

Comment:
- I don't see this as a duplicate, can you please expand on why it's being removed?

Reply:
great, for reference it was this line: `command = cmd + g_ops + [mountpoint]` already adding it